xclim/ensembles/_robustness.py

```diff
@@ -121,28 +126,37 @@ def change_significance(
     ...     delta, test="threshold", abs_thresh=2
     ... )
     """
+    if realization is None:
```

Comment:
I agree with the dummy `realization` dimension method.

However, I think I would prefer to stick to xclim's current specification of  `realization` as the ensemble dimension. All functions in `xclim.ensembles` depend on this syntax. I feel introducing a `realization: str` argument to every one of them is overkill when one only needs to do: `ds.rename(my_ensemble='realization')`. The same way we always expect `time` as the time coordinate.

However, this is only my opinion and I am open to discussion!

Reply:
Agree to stick to the ensemble convention.